composer.json

```diff
@@ -22,8 +22,8 @@
         "php": "^7.2",
 
         "sylius/sylius": "^1.3",
-        "prooph/service-bus-symfony-bundle": "^0.7.0",
-        "knplabs/knp-snappy-bundle": "^1.5"
+        "knplabs/knp-snappy-bundle": "^1.5",
+        "symfony/messenger": "^4.1"
```

Comment:
You also need symfony/framework-bundle ^4.1 to allow for `framework.messenger` configuration.

Reply:
Right 👍